src/components/my-map/my-map.js

```diff
@@ -1,26 +1,29 @@
 import './my-map.scss';
+import './configuration/evt-widget-config-marker-color/evt-widget-config-marker-color';
```

Comment:
I didn't want to group these, i wanted configuration editor to be in its own repo. Despite convention, angular module ids are just strings so you don't have to fake nesting.